language/language_samples.rb

```diff
@@ -21,9 +21,11 @@ def sentiment_from_text project_id:, text_content:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document text_content
```

Comment:
Add a newline after this line. I know it's not part of your change, can you remove extra spaces in front of `language` and `document`.